apicast/src/authorized_callback.lua

```diff
@@ -108,7 +108,7 @@ local function store_code(client_data, params, code)
 end
 
 -- Returns the code to the client
-local function send_code(client_data, params, code)
+local function send_code(client_data, code)
```

Comment:
You are changing the method signature, but not changing the caller. The caller still passes `params` parameter which would get used instead of `code`.